### PR TITLE
Added details for production/post production

### PR DESCRIPTION
**Pull request by RangerMauve**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,20 +17,25 @@
...
 4. If streaming, read questions from chat
 5. Cut-off at 2 hours, closing statements, end stream
 
 ## Post-production
 
-1. Upload the video to the [M3 channel](https://www.youtube.com/channel/UC87Q7_5ooY8FSLwOec52ZPQ) with description
-2. Update the [Schedule](https://github.com/M3-org/schedule)/[Research](https://github.com/M3-org/research) Github repos with timestamps
-3. Finish the meeting and presentation notes on [hackmd.io](https://hackmd.io)
+1. Chop up video into into individual talks
+  - Use [Openshot](https://www.openshot.org/) for editing
```
Comment (avaer) [inline README.md]: FYI I use Premiere. If someone is doing editing work that is a reasonable thing to expense if it's desired.

File: README.md [hunk 2]
```diff
@@ -17,20 +17,25 @@
...
+2. Upload the videos to the [M3 channel](https://www.youtube.com/channel/UC87Q7_5ooY8FSLwOec52ZPQ) with description
+3. Update the [Schedule](https://github.com/M3-org/schedule)/[Research](https://github.com/M3-org/research) Github repos with timestamps
```
Comment (avaer) [inline README.md]: Somewhere it should mention to coordinate some tweeting about it.

We could also do a Youtube premiere.

File: README.md [hunk 3]
```diff
@@ -17,20 +17,25 @@
 3. Make sure speakers are still available and have their slides ready
 4. Send reminders to people as time draws near
```
Comment (avaer) [inline README.md]: Dress rehearsal?

File: README.md [hunk 4]
```diff
@@ -17,20 +17,25 @@
...
 5. (Bonus) Uploading presenter notes to [hackmd.io](https://hackmd.io/cLr7SN-ER2uOvYc_IlC87A)
+6. Double check audio / video quality for folks recording
 
 ## Production
 
 1. Drop into the world, decorate, wait for guests
 2. Start recording/streaming to twitch or youtube
+  - Use [OBS](https://obsproject.com/) for the recording.
+  - Have camera people right click slides to enlarge them periodically
 3. Moderate talks to stay on schedule
```
Comment (avaer) [inline README.md]: Someone should have this as a designated job.

File: README.md [hunk 5]
```diff
@@ -17,20 +17,25 @@
...
 4. If streaming, read questions from chat
```
Comment (avaer) [inline README.md]: Perhaps the moderator could also do this as the job.

**Comment by RangerMauve** [comment]
Should we maybe have a roles list near the top and have some of the stuff that particular roles might want to fulfill?

**Review by avaer — COMMENTED** [review]
We should probably overall have a set of roles/people committing to them.

For example, there is a cameraperson role, event organizer role, timekeeper role...

We've successfully done that ad hoc but it would lend this document structure if it's a list of who does what rather than a list of things someone somewhere has to be on the hook for, and we just pray.

**Comment by RangerMauve** [comment]
How do you feel about having roles assigned in github repos on a per-meetup basis?

Something like a template issue and then a checklist for assigning folks / making sure stuff is done. That's worked out for me for getting tasks done in some of my tech work in the past.

**Comment by avaer** [comment]
That might be best.

File: README.md (hunk 1 above)
Comment (RangerMauve) [reply README.md]: Should I note the expensing in the docs?

File: README.md (hunk 3 above)
Comment (RangerMauve) [reply README.md]: Added in

File: README.md (hunk 2 above)
Comment (RangerMauve) [reply README.md]: Not sure what to put about Youtube Premiere. Just mention it as an option?

File: README.md (hunk 4 above)
Comment (RangerMauve) [reply README.md]: Added!

**Comment by RangerMauve** [comment]
How's this looking? I modified stuff based on comments somewhat and made up some roles.

I'll leave the issue template as a TODO for a separate PR

**Comment by madjin** [comment]
Good stuff :+1:

**Comment by avaer** [comment]
Thanks @RangerMauve!

**Pull request merged**